docs/installation/mac-os.md

```diff
@@ -44,4 +44,14 @@ For additional configuration, [click here](/additional-postgres-setup)
 **Error:** `rbenv: version '<version number>' is not installed (set by /Path/To/Local/Repository/.ruby-version)`
 **_Solution:_** Run the command `rbenv install <version number>`
 
+**Error:** `ruby-build: definition not found: <version number>` when `rbenv` was installed via `brew`.
+
+```bash
+ruby-build: definition not found: <version number>
+
+See all available versions with `rbenv install --list'.                       If the version you need is missing, try upgrading ruby-build:
+```
+**_Solution:_**
+Reinstall `rbenv` by running `brew reinstall rbenv`. After that the version of ruby will be found and installed.
```

Comment:
I usually run what the error message prints out: `brew update && brew upgrade ruby-build`. Just curious, any reason you're recommending `brew reinstall rbenv`?

Reply:
Ahh, I had reinstalled rbenv completely. I think your solution makes more sense.